chef-provisioning-docker.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.homepage = 'https://github.com/opscode/chef-provisioning-docker'
 
   s.add_dependency 'chef'
-  s.add_dependency 'chef-provisioning', '~> 0.9'
+  s.add_dependency 'chef-provisioning', '~> 1.0'
```

Comment:
Agree with @randomcamel and @PierreRambaud on this one. Also, I can confirm that this patch works with chef-provisioning 1.1 at least.